### PR TITLE
Replace calendar with current and past clubs

### PR DESCRIPTION
**Pull request by justinjstark**
Further updates to #29.

- Added current clubs.
- Reversed club timeline to be reverse-chronological.
- Previous clubs are hidden by default but can be shown by clicking "View Previous Clubs".
- Updated club descriptions to remove cascading stories like "the Fuse club continued to...".

I'm not sure of the etiquette for suggesting updates to a previous pull request. I opened this one from a new branch to master to replace #29.

### CODE REVIEW

**Comment by justinjstark** [comment]
This also resolves #9.

**Comment by justinjstark** [comment]
@troynguyen8 Let me know if this one can be merged or if you have suggestions for changes.

File: src/index.html [hunk 1]
```diff
@@ -46,7 +46,7 @@
       <div class="collapse navbar-collapse" id="navbarNav">
         <ul class="navbar-nav ml-auto">
           <li class="nav-item">
-            <a class="nav-link" href="#calendar">Calendar</a>
+            <a class="nav-link" href="#past-clubs">Past Clubs</a>
```
Comment (troynguyen8) [inline src/index.html]: Can you change this to "Our Clubs" to match up with the header? Also change the id for consistency.

File: src/index.html [hunk 2]
```diff
@@ -63,20 +63,55 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
           Girls Code Lincoln was established in January 2016 as Lincoln Girls Who Code. We host technology clubs
           in Lincoln for 4th - 9th grade girls. Since inception we have held over 15 clubs and impacted over 200
           girls.
         </p>
       </section>
 
-      <section id="calendar" class="bg-pink col-12">
-        <iframe src="https://calendar.google.com/calendar/embed?mode=AGENDA&amp;height=400&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=eh2akbgitkihie7k4svqg256u8%40group.calendar.google.com&amp;color=%235229A3&amp;ctz=America%2FChicago"
-          style="border-width:0" width="100%" height="400" frameborder="0" scrolling="no"></iframe>
+      <section id="past-clubs" class="bg-pink col-12 py-5">
```
Comment (troynguyen8) [inline src/index.html]: change id to "our-clubs"

**Comment by troynguyen8** [comment]
Went ahead and implemented the changes I mentioned. Merging now.

File: src/index.html [hunk 3]
```diff
@@ -70,38 +70,54 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
         <h2 class="display3 text-center mb-5">Our Clubs</h2>
         <div class="col-12 d-flex justify-content-center">
-          <div class="col-lg-6 col-xs-8">
+          <div class="col-lg-8 col-xs-10">
             <ul class="timeline">
               <li>
                 <h5 class="timeline-entry-text h5">Spring 2019</h5>
-                <p>Two clubs are being held at FUSE Coworking. The beginner club is learning web development using HTML, CSS, and JavaScript. The intermediate club is focusing on softskills including leadership and mentoring.</p>
+                <p>Two clubs are being held at FUSE Coworking. The beginner club is learning web development using
+                  HTML, CSS, and JavaScript. The intermediate club is focusing on softskills including leadership and
```
Comment (justinjstark) [inline src/index.html]: What's with the crazy line-breaks?

Comment (justinjstark) [reply src/index.html]: nm, saw your comment

**Comment by justinjstark** [comment]
Looks good. Thanks, @Jasbury2 and @troynguyen8.

File: src/index.html [hunk 4]
```diff
@@ -63,20 +63,55 @@ <h5 class="text-light h5 font-weight-bold my-4">Teaching girls about technology
...
-        <iframe src="https://calendar.google.com/calendar/embed?mode=AGENDA&amp;height=400&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=eh2akbgitkihie7k4svqg256u8%40group.calendar.google.com&amp;color=%235229A3&amp;ctz=America%2FChicago"
-          style="border-width:0" width="100%" height="400" frameborder="0" scrolling="no"></iframe>
+      <section id="past-clubs" class="bg-pink col-12 py-5">
+        <h2 class="display3 text-center mb-5">Our Clubs</h2>
+        <div class="col-12 d-flex justify-content-center">
+          <div class="col-lg-6 col-xs-8">
+            <ul class="timeline">
+              <li>
+                <h5 class="timeline-entry-text h5">Spring 2019</h5>
+                <p>Two clubs are being held at FUSE Coworking. The beginner club is learning web development using HTML, CSS, and JavaScript. The intermediate club is focusing on softskills including leadership and mentoring.</p>
```
[the remark this answers is not in the document]
Comment (justinjstark) [reply src/index.html]: Long lines? You gotta use the virtual wrap. :-D

**Pull request merged**